src/detect-engine-analyzer.c

```diff
@@ -1246,7 +1248,7 @@ void EngineAnalysisRules(const DetectEngineCtx *de_ctx,
         if (warn_encoding_norm_http_buf) {
             fprintf(rule_engine_analysis_FD, "    Warning: Rule may contain percent encoded content for a normalized http buffer match.\n");
         }
-        if (warn_tcp_no_flow /*rule_flow == 0 && rule_flow == 0
+        if (warn_tcp_no_flow /*rule_flow == 0 && rule_flags == 0
```

Comment:
What is this change? Is it related to the counting?

Reply:
The comment was incorrect -- it's tracking the code athttps://github.com/OISF/suricata/blob/master/src/detect-engine-analyzer.c#L1128